proposal.rst

```diff
@@ -237,7 +261,115 @@ Project Timeline
 .. any university classes or exams, vacations, etc), make sure you include them
 .. in your project timeline.
 
-FILLME
+
+**April 23 – May 14 2018: Community Bonding Period:**
+
+Expected Deliverables: All working data downloaded, and testing started, with pre-eliminary testing examples and reporting, solving any possible bugs.
+
+	1. Writing tests.
+
+	2. Setting up tools required.
+
+	Also, I would not be able to contribute very actively in the last week due to my end semester exams from May 7th - May 14th.
+
+
+
+**Week 1: May 14 - May 21**
+
+Testing and benchmarking period.
+
+Expected Deliverables: Benchmarked reports for each of the INEX2009 and FIRE dataset. And a report on the running. 
+
+	1. Run benchmarking tests on the data-sets (INEX and FIRE).
+
+	2. Report the evaluation to the mentors. 
+
+	3. Make sure xapian-letor is stable by reporting and fixing bugs if any.
+
+**Week 2: May 14 - May 28**
+
+Testing and benchmarking period.
+
+Expected Deliverables: Stability of xapian-letor stress tested with various dataset examples. 
+
+	1. Writing examples to test proper execution. 
+
+	2. Make sure xapian-letor is stable by reporting and fixing bugs if any.
+
+	3. Cleaning up any previous work. 
+
```

Comment:
Combine the first two weeks. Eliminate blank lines, like the couple below.

Reply:
Did!